src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -215,7 +214,7 @@ class WorkspaceReimburseView extends React.Component {
                         />
                     </Section>
                 )}
-                {Boolean(this.props.hasVBA) && (
+                {!this.props.isLoadingVBA && Boolean(this.props.hasVBA) && (
```

Comment:
`this.props.hasVBA` should be a bool already?
```suggestion
                {!this.props.isLoadingVBA && this.props.hasVBA && (
```